tidb-cloud/backup-and-restore.md

```diff
@@ -96,7 +96,8 @@ To configure the backup schedule, perform the following steps:
 
 > **Note:**
 >
-> The dual region backup feature is currently in beta.
+> - The dual region backup feature is currently in beta.
+> - TiDB Dedicated on Google Cloud Platform works seamlessly with Google Cloud Storage. Similar to Google Cloud Storage, **TiDB Dedicated supports dual-region pairing only within the same multi-region code as Google Dual-region storage**. For example, in Asia, Tokyo and Osaka must currently be paired together for dual-region storage. For more information, refer to [Dual-regions](https://cloud.google.com/storage/docs/locations#location-dr).
```

Comment:
```suggestion
> - TiDB Dedicated clusters hosted on Google Cloud work seamlessly with Google Cloud Storage. Similar to Google Cloud Storage, **TiDB Dedicated supports dual-region pairing only within the same multi-region code as Google dual-region storage**. For example, in Asia, currently you must pair Tokyo and Osaka together for dual-region storage. For more information, refer to [Dual-regions](https://cloud.google.com/storage/docs/locations#location-dr).
```
Google Cloud Platform is now called Google Cloud： https://cloud.google.com/blog/topics/developers-practitioners/introducing-new-homepage-google-cloud